### ISSUE TITLE
Incorrect usage of .toNumber() in groupEvent

### ISSUE THREAD

**Issue by tabam1** [DoctorMcKay/node-steam-user#228]
```
/bot/node_modules/steam-user/components/friends.js:618
		this._emitIdEvent('groupEvent', sid, event.headline, new Date(event.event_time * 1000), event.gid.toString(), event.game_id.toNumber());
		                                                                                                                            ^

TypeError: event.game_id.toNumber is not a function
    at forEach (/bot/node_modules/steam-user/components/friends.js:618:127)
    at Array.forEach (<anonymous>)
    at SteamUser.<anonymous> (/bot/node_modules/steam-user/components/friends.js:601:22)
    at handlers.forEach (/bot/node_modules/steam-user/components/classes/HandlerManager.js:37:12)
    at Array.forEach (<anonymous>)
    at HandlerManager.emit (/bot/node_modules/steam-user/components/classes/HandlerManager.js:36:12)
    at SteamUser._handleMessage (/bot/node_modules/steam-user/components/messages.js:544:24)
    at SteamUser._handleNetMessage (/bot/node_modules/steam-user/components/messages.js:470:7)
    at SteamUser.processMulti (/bot/node_modules/steam-user/components/messages.js:571:9)
    at SteamUser.<anonymous> (/bot/node_modules/steam-user/components/messages.js:565:16)
```

**Comment by chudakM** [comment]
lul i have the same error
are you running your bot like script or it is in electron?

**Comment by tabam1** [comment]
I run with npm forever

**Comment by DoctorMcKay** [comment]
This is actually not a duplicate of #225 as it's happening in another part of the code. This is fixed in v4.3.1.
